content/articles/2018-01-tibble-1-4-2.Rmd

```diff
@@ -0,0 +1,70 @@
+---
+title: tibble 1.4.2
+slug: tibble-1-4-2
+description: >
+    Tibbles are a modern reimagining of the data frame, keeping what time has shown to be effective, and throwing out what is not, with nicer default output too! This article showcases the changes in the most recent version.
+date: 2018-01-24
+author: Kirill Müller, Hadley Wickham
+photo:
+  url: https://unsplash.com/photos/6GjHwABuci4
+  author: Mikael Kristenson
+categories: [package]
+---
+
+
+```{r setup, include = FALSE}
+library(tibble)
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We’re excited to announce version 1.4.2 of the *tibble* package, a minor update focusing mostly on display and performance.  Tibbles are a modern reimagining of the data frame, keeping what time has shown to be effective, and throwing out what is not, with nicer default output too! Grab the latest version with:
+
+```r
+install.packages("tibble")
+```
+
+This release addresses feedback we received for [version 1.4.1](tibble-1-4-1).  We added more options to configure the printing of tibbles, with better defaults. (The option names start with `pillar`, because that's the package that is actually responsible for printing; the new options are available with *pillar* 1.1.0 which will be installed automatically when you install or upgrade *tibble*.)
```

Comment:
This is a small release designed to address the biggest issues introduced in [version 1.4.1](tibble-1-4-1.html). We've introduced a number of option to give you greater control over the appearance, and tweaked the default to work better for more people. (The option names ...